xcube/webapi/context.py

```diff
@@ -461,11 +461,11 @@ def get_dataset_configs_from_stores(self, data_store_pool: DataStorePool) \
                         f'{store_dataset_id}'
                     all_dataset_configs.append(dataset_config)
 
-        # Just for testing:
-        debug_file = 'all_dataset_configs.json'
-        with open(debug_file, 'w') as stream:
-            json.dump(all_dataset_configs, stream)
-            LOG.debug(f'Wrote file {debug_file!r}')
+        # # Just for testing:
+        # debug_file = 'all_dataset_configs.json'
+        # with open(debug_file, 'w') as stream:
+        #     json.dump(all_dataset_configs, stream)
+        #     LOG.debug(f'Wrote file {debug_file!r}')
```

Comment:
Maybe then simply remove? We got version control if you want it back :)

Reply:
No, see, it has a comment. I often need that.

Reply:
Then I simple un-comment it. Our rule is that commented-out code can stay if it has a header comment.